docs/ug/cli.md

```diff
@@ -261,3 +261,40 @@ Cannot be used with any other flags. This flag takes precedence over all other f
   Default: `./reposense-report`
 * Alias: `-v`
 * Example:`--view` or `-v`
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### `--analyze-authorship`, `-A`
+
+**`--analyze-authorship`**: Performs further analysis to distinguish between partial and full credit attribution for
+lines of code assigned to the author.
+
+* Default: this feature is turned ***off*** by default and the author will receive full credit for all lines of code.
+* Alias: `-A` (upper case)
+* Example: `--analyze-authorship` or `-A`
+
+<box type="info" seamless>
+
+A darker background colour represents full credit, while a lighter background colour represents partial credit.
+
+If the code is attributed to a different author by the user via `@@author` tag, then the new author will be given
+partial credit.
+</box>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### `--originality-threshold`, `-ot`
+
+**`--originality-threshold [VALUE]`**: Specifies the cut-off point for partial and full credit
+in `--analyze-authorship`. Author will be given full credit if their contribution exceeds this threshold, else partial
```

Comment:
Ideally, we may want to explain how "contribution" is computed here, perhaps in a tooltip too. But if it's a complicated calculation, it's better to avoid it.

On a further note, we should add how authorship is analysed to the developer docs in another PR.

Reply:
Added a brief explaination about this.